doc/swish/http.tex

```diff
@@ -273,10 +416,11 @@
 \returns{} a string \alt{} \code{\#f}
 
 The \code{http:find-param} procedure returns the value associated with
-\var{name} in \var{params} if present and \code{\#f}
-otherwise. Parameter comparisons are case-sensitive. If \var{name} is
-not a string, exception \code{\#(bad-arg http:find-param \var{name})}
-is raised.
+\var{name} in \var{params} if present and \code{\#f} otherwise.  If
+\var{name} is a symbol, the lookup uses \code{eq?}.  If \var{name} is
+a string, it is first mapped to a lower-case symbol before performing
```

Comment:
http:find-param does a case-sensitive comparison, so the string isn't downcased.